lib/shim/datastore-shim.js

```diff
@@ -77,24 +77,17 @@ const QUERY_PARSERS = {
  *  A helper class for wrapping datastore modules.
  * @param {Agent} agent
  *  The agent this shim will use.
+ * @param shimName
  * @param {string} moduleName
  *  The name of the module being instrumented.
  * @param {string} resolvedName
  *  The full path to the loaded module.
- * @param {string} [datastoreId]
- *  The name of the datastore being instrumented. If available, use one of the
- *  values from {@link DatastoreShim.DATASTORE_NAMES}.
- *  Calls {@link DatastoreShim#setDatastore} if datastoreId is
- *  specified.
  * @see Shim
  * @see DatastoreShim.DATASTORE_NAMES
  */
-function DatastoreShim(agent, moduleName, resolvedName, datastoreId) {
```

Comment:
this was done for testing. customers really couldn't do this with our APIs, I removed because it was unused and I added another arg